crib/scripts/check_env_vars.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
We shouldn't need this if we just add them to the .env, right?

See: https://github.com/smartcontractkit/ccip/pull/731/files#diff-56634260dc609e181d0119332f1bd3342b2a602893af3f8b0ffcedac03dbe851R5 which gets included here https://github.com/smartcontractkit/ccip/blob/d8a58d3dde7972c1aea11b674ffa0e9869a55054/crib/devspace.yaml#L5